pkg/lockfile/ecosystems.go

```diff
@@ -1,5 +1,6 @@
 package lockfile
 
+// Returns a slice of all known ecosystems, matching the results in pkg/models.
```

Comment:
This comment should explain the actual relevance of the method, rather than just what it technically returns:

```suggestion
// KnownEcosystems returns a list of ecosystems that `lockfile` supports automatically inferring an extractor for based on a file path
```

(I really wish I'd named this something different now 😅)